docs/high-availability/provision-flannel.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function setup_flannel {
+    yum install -y flannel
+
+    ### Write this k/v to etcd.  Flannel will grab it to setup its networking.
+    curl --silent -s -L http://kube0.ha:2379/v2/keys/coreos.com/network/config -XPUT -d value='{"Network": "80.12.13.14/8", "SubnetLen": 24, "Backend": {"Type": "vxlan"}}'
```

Comment:
I think my comment got swallowed. This should almost surely be a private IP range? It's also huge. I won't be able to talk to any 80.* IPs. This would provide 8 nodes IPs for 32 pods:
`{"Network": "172.31.255.0/24","SubnetLen":27}`